scripts/build/libc/newlib.sh

```diff
@@ -125,16 +128,18 @@ ENABLE_TARGET_OPTSPACE:target-optspace
     #   build  : not used
     #   host   : the machine building newlib
     #   target : the machine newlib runs on
-    CT_DoExecLog CFG                                    \
-    CC_FOR_BUILD="${CT_BUILD}-gcc"                      \
-    CFLAGS_FOR_TARGET="${cflags_for_target}"            \
-    AR=${CT_TARGET}-ar                                  \
-    RANLIB=${CT_TARGET}-ranlib                          \
-    "${CT_SRC_DIR}/newlib-${CT_LIBC_VERSION}/configure" \
-        --host=${CT_BUILD}                              \
-        --target=${CT_TARGET}                           \
-        --prefix=${CT_PREFIX_DIR}                       \
-        "${newlib_opts[@]}"                             \
+    CT_DoExecLog CFG                                               \
+    CC_FOR_BUILD="${CT_BUILD}-gcc"                                 \
+    CFLAGS_FOR_TARGET="${cflags_for_target}"                       \
+    AR_FOR_TARGET="`which ${CT_TARGET}-gcc-ar`"                    \
+    RANLIB_FOR_TARGET="`which ${CT_TARGET}-gcc-ranlib`"            \
+    ac_cv_path_AR_FOR_TARGET="`which ${CT_TARGET}-gcc-ar`"         \
+    ac_cv_path_RANLIB_FOR_TARGET="`which ${CT_TARGET}-gcc-ranlib`" \
```

Comment:
Can you explain why both `AR_FOR_TARGET` and `ac_cv_path_AR_FOR_TARGET` are needed? Is it for different versions of newlib, or does newlib indeed require both?

In any case, a comment with an explanation is warranted.